sql-cli/sql_cli/__main__.py

```diff
@@ -37,111 +41,99 @@ def about() -> None:
+        dags_directory=project.airflow_dags_folder,
+    )
     rprint("The DAG file", dag_file.resolve(), "has been successfully generated. 🎉")
 
 
 @app.command(
-    help="Validate Airflow connection(s) provided in the configuration file for the given environment."
+    help="""
+    Validate Airflow connection(s) provided in the configuration file for the given environment.
+    """
 )
 def validate(
-    environment: str = typer.Argument(
+    project_dir: Path = typer.Argument(
+        None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
+    ),
+    env: str = typer.Option(
         default="default",
-        help="environment to validate",
+        help="(Optional) Environment used to declare the connections to be validated",
     ),
-    connection: str = typer.Argument(
+    connection: str = typer.Option(
         default=None,
-        help="airflow connection id to validate",
+        help="(Optional) Identifier of the connection to be validated. By default checks all the env connections.",
     ),
 ) -> None:
-    validate_connections(environment=environment, connection_id=connection)
+    project_dir_absolute = project_dir.resolve() if project_dir else Path.cwd()
+    project = Project(project_dir_absolute)
+
+    validate_connections(project=project, environment=env, connection_id=connection)
 
 
 @app.command(
     help="""
     Run a workflow locally. This task assumes that there is a local airflow DB (can be a SQLite file), that has been
-    initialized with Airflow tables. Users can also add paths to a connections or variable yaml file which will override
-    existing connections for the test run.
-
-    \b
-    Example of a connections.yaml file:
-
-    \b
-    my_sqlite_conn:
-        conn_id: my_sqlite_conn
-        conn_type: sqlite
-        host: ...
-    my_postgres_conn:
-        conn_id: my_postgres_conn
-        conn_type: postgres
-        ...
+    initialized with Airflow tables.
     """
 )
 def run(
-    directory: Path = typer.Argument(
+    workflow_name: str = typer.Argument(
         default=...,
         show_default=False,
-        exists=True,
-        help="directory containing the sql files.",
+        help="name of the workflow directory within workflows directory.",
     ),
-    dags_directory: Path = typer.Argument(
-        default=...,
-        show_default=False,
-        exists=True,
-        help="directory for the generated DAGs.",
-    ),
-    connection_file: Path = typer.Option(
-        default=None,
-        exists=True,
-        help="path to connections yaml or json file",
+    env: str = typer.Option(
+        metavar="environment",
+        default="default",
+        help="environment to run in",
     ),
-    variable_file: Path = typer.Option(
-        default=None,
-        exists=True,
-        help="path to variables yaml or json file",
+    project_dir: Path = typer.Option(
```

Comment:
At the moment the run command prints something like:
```
Created dagrun <DagRun example_templating @ 
2022-10-19T09:56:48.703948+00:00: 
manual__2022-10-19T09:56:48.703948+00:00, state:running, queued_at: None.
externally triggered: False> <DagRun example_templating @ 
2022-10-19T09:56:48.703948+00:00: 
manual__2022-10-19T09:56:48.703948+00:00, state:running, queued_at: None.
externally triggered: False>
*****************************************************
Running task filtered_orders
filtered_orders ran successfully!
*****************************************************
*****************************************************
Running task joint_orders_customers
joint_orders_customers ran successfully!
*****************************************************
Dagrun example_templating final state: success
```

This is significantly different from what we had originally specced:
https://github.com/astronomer/astro-sdk/issues/942

I suggest we have a ticket to address this and try to complete it before 0.1